manifests/Quazar Conundrum.yaml

```
@@ -0,0 +1,11 @@
+name: The Quazar Condundrom
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/releases/tag/Major_ReleaseV015
```

This should point to your github repo, though
```suggestion
homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/
```